test/unit/Pool/removeAsset.js

```diff
@@ -159,6 +159,7 @@ describe('removeAsset', function () {
       assert.strictEqual(minAmount.toString(), ether('24360').toString());
       assert.strictEqual(maxAmount.toString(), ether('32500').toString());
       assert.strictEqual(maxSlippageRatio.toString(), '0');
+      // TODO: does this need to be exactly this timestamp?
       assert.strictEqual(lastSwapTime.toString(), '1633425218');
```

Comment:
The `lastSwapTime` is no longer manually set in the constructor, but instead goes through `addAsset()`, which sets lastSwapTIme to 0. I didn't make any changes for this as I wanted to ask if this exact timestamp is still required?